internal/resources/storage_statefulset.go

```diff
@@ -248,9 +315,29 @@ func (b *StorageStatefulSetBuilder) buildVolumeMounts() []corev1.VolumeMount {
 		})
 	}
 
+	if b.Spec.CaBundle != "" {
+		volumeMounts = append(volumeMounts, corev1.VolumeMount{
+			Name:      initMainSharedCertsVolumeName,
+			ReadOnly:  true,
+			MountPath: systemSslStorePath,
+		})
+	}
+
 	return volumeMounts
 }
 
+func (b *StorageStatefulSetBuilder) buildInitContainerArgs() ([]string, []string) {
+	command := []string{"/bin/bash", "-c"}
+
+	args := []string{fmt.Sprintf("%s && %s && %s",
+		"apt-get update",
+		"apt-get install -y ca-certificates",
```

Comment:
It should not be necessary to install `ca-certificates`, it should be installed to the docker image of YDB at build time. We're now discussing whether it can be done with no breaking changes.